### ISSUE TITLE
librarycarpentry/lc-sql

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#69]
tracking issue for <https://github.com/librarycarpentry/lc-sql>

 - [x] fix asset image links
 - [x] fix swcarpentry links
 - [x] fix external links that should be internal (e.g. https://github.com/LibraryCarpentry/lc-sql/blob/aa9e12ab9c1c76c00b4e2efd80f6fa85469391de/_extras/guide.md?plain=1#L38)
 - [x] fix asset image lnks that just won't die because it's an HTML link 😮‍💨 
 - [x] move learner profile to the `profiles/` folder 🎉 

```
episodes/06-joins-aliases.md:39 [missing file]: [](assets/fig/left-join-articles-journals_40.png)
episodes/06-joins-aliases.md:51 [missing file]: [](assets/fig/articles-erd-v02.png)
episodes/08-database-design.md:31 [missing file]: [](assets/fig/doaj-spreadsheet.png)
episodes/08-database-design.md:57 [missing file]: [](assets/img/field-record-value.png)
episodes/08-database-design.md:100 [missing file]: [](assets/fig/normalisation.png)
instructors/instructor-notes.md:69 [link text too short]: [R](https://swcarpentry.github.io/sql-novice-survey/11-prog-R/index.html)
```

**Comment by jyssy** [comment]
@tobyhodges and @zkamvar , the LC-SQL beta lesson is looking good. I like the style changes very much.

**Comment by zkamvar** [comment]
This run was successful, but had issues as described in https://github.com/carpentries/lesson-transition/commit/a359d3eddba2ddeb5a6c2394455d1bebb4570de1#commitcomment-111506914

**Comment by zkamvar** [comment]
fixed in a359d3eddba2ddeb5a6c2394455d1bebb4570de1
